### ISSUE TITLE
Where is the validation set?

### ISSUE THREAD

**Issue by JACOBWHY** [PJLallen/OSFormer#19]
The paper states that "OSFormer requires only about 3,000 images to train, and it converges rapidly." I noticed that OSFormer seems to have no validation set during training(just saving the last model_089999.pth), so how can we determine network convergence and reach the optimal? Thank you for your reply.
